app/Http/Controllers/ActivityController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Activity;
+use Illuminate\Http\Request;
+use App\User;
+use Auth;
+use File;
+use Image;
+use Validator;
+use Gate;
+
+class ActivityController extends Controller
+{
+        /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // get user
+        $user = Auth::user();
```

Comment:
Initialisation of variables should be separate by a white line with the next lines of code